src/pages/404.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+const NotFound = () => {
+  const router = useRouter();
+  useEffect(() => {
+    setTimeout(() => {
+      router.push('/');
```

Comment:
this seems odd from UX => show 404 and then after 50 secs to redirect ?
I would propose either do a redirection OR 404 with some links within page (Home, Projects ...)